src/components/role-mapping/RoleMapping.tsx

```diff
@@ -172,12 +174,16 @@ export const RoleMapping = ({
   const [hide, setHide] = useState(false);
   const [showAssign, setShowAssign] = useState(false);
   const [selected, setSelected] = useState<Row[]>([]);
+  const [wait, toggle] = useToggle();
```

Comment:
Might be worth it to add what is being toggled to the name.
```suggestion
  const [wait, toggleWait] = useToggle();
```